spec/features/services/list_collection_associations_spec.rb

```diff
@@ -76,8 +76,8 @@
 
       it 'displays the existing collection assiciations' do
         expect(page).to have_content('Disabled rows')
-        expect(page).to have_selector('tbody tr', count: 28)
-        expect(page).to have_selector('#selected_collections_', count: 2)
+        expect(page).to have_selector('tbody tr', count: 29)
+        expect(page).to have_selector('#selected_collections_', count: 3)
```

Comment:
Took me way longer than it should have to figure out why this was increased. In this case, the selected collections are actually the selectable collections because the setup for this tests assigns 26 collections to the service.